content/docs/5.1/deploy/operate/scale.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Scale
+description: Learn how to scale out/in TiKV using TiUP
+menu:
+    "5.1":
+        parent: Operate TiKV
+        weight: 2
+---
+
+The capacity of a TiKV cluster can be increased or decreased without interrupting the online services.
+
+This document describes how to scale the TiKV and PD using TiUP.
+
```

Comment:
```suggestion

## View a TiKV cluster

```

Reply:
`View a TiKV cluster` is not a chapter of this page.
It is mentioned in case the user doesn't know how the cluster name.